src/sidebar/helpers/mentions.ts

```diff
@@ -77,9 +77,6 @@ export function wrapDisplayNameMentions(
     DISPLAY_NAME_MENTIONS_PAT,
     (match, precedingChar, displayName) => {
       const suggestion = usersMap.get(displayName);
-
-      // TODO Should we still build a mention tag so that it renders as an
-      //      invalid mention instead of plain text?
```

Comment:
This comment references a discussion we already had, and decided to keep it as plain text for now.